Automattic/jetpack
# Jetpack Settings: WAF standalone mode indicator

### Description
Adds an indicator within the Firewall settings when standalone mode is enabled.

Related Protect PR: https://github.com/Automattic/jetpack/pull/34840

**Note**: I've used a common `Status` component with a `label={ __( Standalone mode, slug ) }` as the indicator across both plugins but I am open to suggestions as to other, more attractive or functional ways of doing so.

## Proposed changes:

* Add a condition to the Firewall settings header to indicate when the mode is active

### Other information:

## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/893

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch
* Start your Jurassic Tube
* Enable Jetpack and verify that the WAF is enabled from the Firewall section on the Settings page
* Create a `.user.ini` file in `tools/docker/wordpress` and add the following directive: `auto_prepend_file = "/var/www/html/wp-content/jetpack-waf/bootstrap.php"`
* Return to the Firewall settings and confirm that the Standalone mode indicator is now visible in the Firewall section header

## Screenshots
WAF active, Standalone mode disabled:
<img width="1044" alt="Screen Shot 2024-01-04 at 12 16 30" src="https://github.com/Automattic/jetpack/assets/43220201/e22ea292-8e4a-4cdc-b121-faf3f5e6baaf">

WAF active, Standalone mode enabled:
<img width="1040" alt="Screen Shot 2024-01-04 at 12 17 21" src="https://github.com/Automattic/jetpack/assets/43220201/0fcf1b2c-f5b4-46ac-96f9-ff5e8a0a89c5">

WAF inactive:
<img width="1042" alt="Screen Shot 2024-01-04 at 12 15 52" src="https://github.com/Automattic/jetpack/assets/43220201/5bdba6bd-d7ad-42d0-af19-b5600e1781a8">
